pom.xml

```diff
@@ -691,43 +728,26 @@
                     <version>2.9</version>
                 </plugin>
 
-                <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>animal-sniffer-maven-plugin</artifactId>
-                    <version>1.15</version>
-                    <executions>
-                        <execution>
-                            <id>check-jdk6</id>
-                            <phase>process-classes</phase>
-                            <goals>
-                                <goal>check</goal>
-                            </goals>
-                            <configuration>
-                                <signature>
-                                    <groupId>org.codehaus.mojo.signature</groupId>
-                                    <artifactId>java16</artifactId>
-                                    <version>1.0</version>
-                                </signature>
-                                <annotations>
-                                    <annotation>com.datastax.driver.core.IgnoreJDK6Requirement</annotation>
-                                </annotations>
-                            </configuration>
-                        </execution>
```

Comment:
Deletion of the above is explained further above - it's OpenTelemetry compatibility matter.